eva/models/storage/batch.py

```diff
@@ -378,7 +380,14 @@ def aggregate(self, method: str) -> None:
         Arguments:
             method: string with one of the five above options
         """
-        self._frames = self._frames.agg([method])
+        # Aggregate ndarray
+        if isinstance(self._frames.iat[0, 0], np.ndarray):
```

Comment:
Is it aggregating each row of the array? If yes, I suspect that will break the execution logic.

Reply:
Yes, how will it break the execution logic?

The NDARRAY case is for object detection array etc. The normal case is the one that existed earlier -- `self._frames = self._frames.agg([method])`

Reply:
I have reverted back the `NDARRAY` case as it does not make sense.

Reply:
Aggregate on `ndarray` and primitive column will result in different row counts. 

We have a different set of aggregate operators to apply row-wise aggregates like`Array_Count`